.github/dependabot.yml

```diff
@@ -96,6 +96,22 @@ updates:
       - dependency-name: org.wiremock:wiremock
       - dependency-name: org.wiremock:wiremock-standalone
       - dependency-name: uk.co.automatictester:wiremock-maven-plugin
+    ignore:
+      # Major/minor upgrades require more work and synchronization, we do them manually.
+      # Only use dependabot for micros (patch versions).
+      - dependency-name: org.hibernate.*:*
+        update-types: ["version-update:semver-major", "version-update:semver-minor"]
```

Comment:
I think this line needs to be removed?

Reply:
Why? Doesn't it apply to [lines 66-70](https://github.com/quarkusio/quarkus/pull/40276/files#diff-dd4fbda47e51f1e35defb9275a9cd9c212ecde0b870cba89ddaaae65c5f3cd28R66-R70) above?

Reply:
Ah my bad, didn't know you could do that in the `ignore`: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#ignore